src/components/Comments/Comments.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react'
+import { CommentCard } from '../CommentCard/CommentCard'
+import { getCommentsById } from '../../../APIs'
+
+export const Comments = ({ article_id }) => {
+
+    const [comments, setComments] = useState([])
+    useEffect(() => {
+        getCommentsById(article_id)
+            .then((comments) => {
+                setComments(comments)
+            })
+    }, [article_id])
+
+
+    return (
+        <>
+            <section>
+                {comments.map((comment) => {
```

Comment:
Would be a good idea to conditionally render something like a simple p tag stating "no comments exist for this article yet" for a given article that exists but has no comments (article 37 if you want to see what that looks like right now)